templates/web/highwaysengland/report/new/inline-tips.html

```diff
@@ -0,0 +1,11 @@
+<label for="where_hear">How did you hear about us?</label>
+<select class="form-control" name="where_hear" id="where_hear">
+<option value="">[% loc('-- Pick an option --') %]</option>
+<option value="Facebook" [% IF 'Facebook' == report_meta.where_hear.value %]selected[% END %]>Facebook</option>
```

Comment:
So I don't think you can use report_meta here either - will just have to store where_hear=report.get_extra_metadata('where_hear') looking at how sortmysign does it